components/brave_wallet/renderer/js_cardano_wallet_api.cc

```diff
@@ -28,6 +28,67 @@
 
 namespace brave_wallet {
 
+namespace {
+
+// See Error Types section from https://cips.cardano.org/cip/CIP-30
+int ConvertErrorCode(const mojom::CardanoProviderErrorCodes code) {
+  switch (code) {
+    // Api error
+    case mojom::CardanoProviderErrorCodes::kAPIErrorInvalidRequest:
+      return -1;
```

Comment:
let's give values to mojm enum entries like kAPIErrorInvalidRequest = -1 etc
so we don't need ConvertErrorCode